app/components/view_component/form/check_box_component.rb

```diff
@@ -0,0 +1,47 @@
+# frozen_string_literal: true
+
+module ViewComponent
+  module Form
+    class CheckBoxComponent < FieldComponent
+      attr_reader :checked_value, :unchecked_value
+
+      def initialize(form, object_name, method_name, checked_value, unchecked_value, options = {}) # rubocop:disable Metrics/ParameterLists
+        @checked_value   = checked_value
+        @unchecked_value = unchecked_value
+
+        super(form, object_name, method_name, options)
+      end
+
+      def call # rubocop:disable Metrics/AbcSize
+        if options[:label] == false
```

Comment:
Do we keep this `label` option or do we stick to the [default Rails features](https://api.rubyonrails.org/classes/ActionView/Helpers/FormBuilder.html#method-i-check_box)?

Reply:
Good question

Reply:
I would say let's keep it simple first: implement exactly the Rails features, and create issues to take note of what other options could be added later.

Also, I think that it's probably better to reuse the future "group" component for this instead of including it in the check box component.

Reply:
I removed it for now, and opened #16